docs/source/examples/Output Widget.ipynb

```diff
@@ -153,7 +153,11 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
```

Comment:
What does this do?

Reply:
https://nbsphinx.readthedocs.io/en/0.5.0/allow-errors-per-cell.html